docs/UserGuide.md

```diff
@@ -55,8 +62,11 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `addR n/salad`.
+* All commands are case-sensitive.<br>
+  e.g. in `addR`, `add` is in small letters while `R` is in capital letters.
+
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. in `addR n/NAME`, `NAME` is a parameter which can be used as `addR n/salad`.
 
 * Items in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/Salad t/healthy` or as `n/Salad`.
```

Comment:
Items with …​ after them can be used more than 1 times.
e.g. [t/TAG]…​ can be used as t/healthy, t/healthy t/low calories etc.

Not sure why this is not shown here but can change the ',' after t/healthy to 'or' so its clearer?

Reply:
ok good idea